x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -9624,578 +9624,6 @@
     "xpack.infra.waffle.unableToSelectMetricErrorTitle": "メトリックのオプションまたは値を選択できません",
     "xpack.infra.waffleTime.autoRefreshButtonLabel": "自動更新",
     "xpack.infra.waffleTime.stopRefreshingButtonLabel": "更新中止",
-    "xpack.ingestManager.agentBulkActions.agentsSelected": "{count, plural, one {#個のエージェント} other {#個のエージェント}}が選択されました",
```

Comment:
I think it would be nice to rename all the keys manually in this file and `zh-CN` to avoid translators having to re-translate all the text. what do you think?

also note that this file is alphabetical by key so `xpack.fleet` would be higher up

Reply:
Sure 👍